solidity/migrations/2_deploy_contracts.js

```diff
@@ -57,6 +58,15 @@ const all = [
 
 module.exports = (deployer, network, accounts) => {
   deployer.then(async () => {
+    let constantsContract = TBTCConstants
+    if (network == "keep_dev" || network == "development") {
+      // For keep_dev and development, replace constants with testnet constants.
+      // Masquerade as TBTCConstants like a sinister fellow.
```

Comment:
👻

Reply:
Still don't quite understand why we're adding it to the `all` array @Shadowfiend. AFAIK the *TBTCConstants library doesn't need anything linked into it.

Reply:
I think I added this back when I misunderstood the reason for the issue that led me to hardcore the contract name here. I’ll see about pulling it back out.